doc/optimizations.md

```diff
@@ -23,12 +30,12 @@ This is usually done by symlinking them from their original location (something
 eg:
 ```
 cd /var/lib/tomcat-geoserver0/lib
-sudo ln -s /usr/share/java/jai_core.jar .
-sudo ln -s /usr/share/java/jai_codec.jar .
-sudo ln -s /usr/share/java/mlibwrapper_jai.jar .
-sudo ln -s /usr/share/java/jai_imageio.jar .
-sudo ln -s /usr/share/java/clibwrapper_jiio.jar .
-sudo service tomcat-geoserver0 restart
+ln -s /usr/share/java/jai_core.jar .
+ln -s /usr/share/java/jai_codec.jar .
+ln -s /usr/share/java/mlibwrapper_jai.jar .
+ln -s /usr/share/java/jai_imageio.jar .
+ln -s /usr/share/java/clibwrapper_jiio.jar .
+service tomcat-geoserver0 restart
```

Comment:
I miss one sudo in my correction. I remove them because 'sudo' is used in Ubuntu, not in debian by default.